Octokit.Tests.Integration/Clients/RepositoryBranchesClientTests.cs

```diff
@@ -848,6 +846,143 @@ public TheDeleteRequiredStatusChecksContextsMethod()
         }
     }
 
+    public class TheGetAdminEnforcementMethod : IDisposable
+    {
+        private readonly IRepositoryBranchesClient _client;
+        private readonly RepositoryContext _userRepoContext;
+
+        public TheGetAdminEnforcementMethod()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            _client = github.Repository.Branch;
+
+            _userRepoContext = github.CreateRepositoryWithProtectedBranch().Result;
+        }
+
+        [IntegrationTest]
+        public async Task GetAdminEnforcement()
```

Comment:
Minor quibble but the method names of the tests should be worded as actions, eg Get**s**AdminEnforcement rather than GetAdminEnforcement.  
This is so that the fully qualified test case names look like:  `TheGetAdminEnforcementMethod.GetsAdminEnforcement`